docs/_articles/en/v3.x/architecture/pmd-engine.md

```diff
@@ -13,9 +13,31 @@ You can change which rules are executed by using the flags described in the Code
 
 Refer to our [FAQ](./en/v3.x/faq/#q-how-do-i-enable-engine-xs-default-rules-for-language-y) for info on how to enable PMD’s built-in rules for other languages.
 
+## How do I use pmd-appexchange to prepare my solution for an AppExchange security review?#
+
+In addition to the base PMD engine, Code Analyzer also includes a custom PMD variant, `pmd-appexchange`. The rules included in `pmd-appexchange` help AppExchange partners prepare their managed packages for security review.
+
+The `pmd-appexchange` engine is disabled by default. To run a PMD scan with the AppExchange-specific ruleset, run `sf scanner run` with the `--engine pmd-appexchange` flag.
+
+**Example:**
+
+`sf scanner run --engine pmd-appexchange --target ./`
+
+For more information on the `pmd-appexchange` rules, read the [pmd-appexchange command reference](https://github.com/forcedotcom/sfdx-scanner/tree/dev/pmd-appexchange).
+
+If you’re an AppExchange partner submitting your managed package for security review, you must scan it with Salesforce Code Analyzer and provide test results in your solution’s AppExchange Security Review submission. To run the required PMD scan with the AppExchange-specific ruleset, and produce the required scan report, run `sf scanner run --engine pmd-appexchange`, and name the output file CodeAnalyzerPmdAppExchange.csv.
```

Comment:
Lines 28-34 of this page contain a lot of the same information as the ISVforce Guide page, which means we'd have to make sure they stay in sync if anything ever changes. Should we delete 28-32 and change 34 similar to the other suggestion for the release notes?